src/main/java/ja/workshops/spring/części/konfiguracja/adnotacjami/a_adnotacje_wstrzykiwania/autowired/Main.java

```diff
@@ -0,0 +1,29 @@
+package ja.workshops.spring.części.konfiguracja.adnotacjami.a_adnotacje_wstrzykiwania.autowired;
+
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+/**
+ * Tworzenie {@link ApplicationContext kontekstu aplikacji} springowej.
+ *
+ * @author Wiktor Rup
+ */
+class Main {
+
+    public static void main(String[] args) {
+
+
+        ApplicationContext kontekst = new ClassPathXmlApplicationContext("AutowiredBeans.xml");
```

Comment:
:-1: brak wzmianki o tym kontekście, a jest to diablo istotny element całej sprawy. Co takiego robi `ClassPathXML...`? Dlaczego go używamy? Owszem, w następnym pakiecie wzmianka jest, ale tutaj nie widzę powodu by to omijać. Jeśli faktycznie chcemy to omijać na potrzeby warsztatu, poproszę o zdanie tego dotyczące.